models/DataObject/Data/QuantityValueRange.php

```diff
@@ -64,12 +64,7 @@ public function setMaximum(int|float|null $maximum): void
         $this->markMeDirty();
     }
 
-    public function getRange(int $step = 1): array
-    {
-        return range($this->getMinimum(), $this->getMaximum(), $step);
-    }
-
-    public function getValue(int $step = 1): array
+    public function getValue(int|float $step = 1): array
     {
         return $this->getRange($step);
```

Comment:
```suggestion
        return [$this->getMinimum(), $this->getMaximum()];
```
@QuantumRange We think `getValue` should return the saved values, otherwise this function just calls the `getRange` function